stock_picking_show_return/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
More a question out of curiosity: our template states a header with license+copyright but a lot of people do not include one for the init.py file. In your case it is incomplete (no copyright).
I never block for this but am curious about what should be enforced...